.circleci/config.yml

```diff
@@ -2,13 +2,42 @@ version: 2.1
 
 orbs:
   php: circleci/php@1.0
-  wp-svn: studiopress/wp-svn@0.1
+  wp-svn: studiopress/wp-svn@0.2
 
+references:
+  PLUGIN_SLUG: &PLUGIN_SLUG
+    genesis-simple-hooks
+
+jobs:
+  zip:
```

Comment:
This `.zip` job lets us test the wp.org deployment without doing the deployment.

It uses the same `.distignore`, so we can see if that allows the files it should, and ignores the files it should.